bundles/af-core/src/test/resources/form/hcaptchav2/exporter-hcaptcha.json

```diff
@@ -0,0 +1,38 @@
+{
+    "id": "hcaptcha-abc574167c",
+    "fieldType": "captcha",
+    "name": "test-hcaptcha",
+    "visible": true,
+    "type": "string",
+    "required": true,
+    "enabled": true,
+    "readOnly": false,
+    "captchaDisplayMode": "invisible",
+    "captchaProvider": "hcaptcha",
+    "captchaSiteKey": "dummySiteKey",
+    "label": {
+      "visible": true,
+      "value": "HCAPTCHA"
+    },
+    "events": {
+      "custom:setProperty": [
+        "$event.payload"
+      ]
+    },
+    "properties": {
+      "fd:dor": {
+        "dorExclusion": false
+      },
+      "fd:path": "/content/hcaptcha",
+      "fd:captcha": {
```

Comment:
In the new way, why do we have still have fd:captcha, is this required ?

Reply:
We still need some custom properties required at the client side that are not covered by the props we added at the top level.